website/docs/d/maintenance_window.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: maintenance_window"
+sidebar_current: "docs-mongodbatlas-datasource-maintenance_window"
+description: |-
+    Provides a Maintenance Window Datasource.
+---
+
+# mongodbatlas_maintenance_window
+
+`mongodbatlas_maintenance_window` provides a Maintenance Window entry datasource. Gets scheduled maintenance event for a project.
```

Comment:
slight edit: provides a Maintenance Window entry datasource. Gets information regarding the configured maintenance window for a MongoDB Atlas project.